server/repository/migrations/postgres/2021-12-10T10-00_number_table/up.sql

```diff
@@ -8,8 +8,7 @@ CREATE TYPE number_type AS ENUM (
 );
 
 -- Numbering table holding a list of typed counters
-CREATE TABLE
-  number (
```

Comment:
I would not touch these migration files. Migrations are read only once they are live. Sometimes you want to check that nothing has changed in your current branch in old migrations. Doing these style fixes can add some unnecessary noise...

Reply:
Sorry, just made it easier for me to search and doesn't affect migration...